test/unit/connectors/SubscriptionDisplayConnectorSpec.scala

```diff
@@ -51,15 +51,15 @@ class SubscriptionDisplayConnectorSpec extends BaseSpec {
     "return None when no EORINo received from subscription display" in {
       val responseBody = Json.parse("""{"subscriptionDisplayResponse": {}}""")
       when(mockHttp.doGet(any(), any())(any(), any())).thenReturn(
-        Future.successful(HttpResponse(200, Some(responseBody)))
+        Future.successful(HttpResponse(status = 200, json = responseBody, headers = Map.empty))
```

Comment:
Here two methods has the same type of arguments, only different argument names, need to be defined explicit